spec/lucky/file_response_spec.cr

```diff
@@ -0,0 +1,101 @@
+require "../spec_helper"
+
+include ContextHelper
+
+describe Lucky::FileResponse do
+  describe "#print" do
+    describe "file is missing" do
+      it "raises an exception" do
+        context = build_context
+
+        expect_raises Lucky::Exceptions::MissingFile, /^Cannot read file/ do
+          print_file_response(context, "nope")
+        end
+      end
+    end
+
+    describe "status_code" do
+      it "uses the default status if none is set" do
+        context = build_context
+        print_file_response(context, "lucky_logo.png")
+        context.response.status_code.should eq Lucky::TextResponse::DEFAULT_STATUS
```

Comment:
Could you add newlines between setup/exercise/verify? I think this is plenty readable as-is, but adding the newlines would fit in with what is done in most of the other specs